### ISSUE TITLE
New version 17/05

### ISSUE THREAD

**Issue by Lofty73** [jnthas/clockwise#26]
I am encountering problems still with the Access point on the clocks. I finally got the Pac Man clock with the shortened days but now the web interface seems broken. I download the file and eventually get it to accept my wifi password but it say connected but the display does not reset. I cannot get the web page to display and when i reset my screen it seems to log onto my network and hang on NTP server.
I have tried different downloads and two different ESP devices.

**Comment by jnthas** [comment]
Hi Nigel, I merged a pull request right now that can be useful for you. It will be possible to change the NTP server in the settings page, I did some tests this weekend and there are some servers way faster than the default we are using. For example, the google one. I let you know when I release the version, probably at the end of the week. 
https://gist.github.com/mutin-sa/eea1c396b1e610a2da1e5550d94b0453
